blocks/log.py

```diff
@@ -221,6 +227,9 @@ def _to_dataframe(self):
 
 class TrainingStatus(AbstractTrainingStatus):
     """A simple training status."""
+    def __getattr__(self, key):
```

Comment:
Yes, I know it looks strange, but at least it is consistent with the current log behaviour. :)